components/resolved_ts/src/resolver.rs

```diff
@@ -169,6 +200,19 @@ impl Resolver {
                 .inc();
         }
 
+        let min_raw_ts = self
```

Comment:
Should be placed before `if self.resolved_ts >= new_resolved_ts {` to make `RTS_RESOLVED_FAIL_ADVANCE_VEC` metrics correct.

Suggest to add metrics for the force untrack.
Suggest to make `RAW_LOCK_TS_TTL` configurable.